src/app/give-help/accommodation-form/accommodation-form.component.ts

```diff
@@ -22,13 +22,22 @@ export class AccommodationFormComponent {
     hostLanguage: [],
   });
   loading: boolean = false;
+  @ViewChild('phoneInput') phoneInput!: { nativeElement: { value: any } };
```

Comment:
Kontynuując z https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/pull/135#discussion_r831883221:
> Dodaj proszę poprawny typ do nativeElement zamiast tworzyć swój. Cały DOM jest otypowany w TS.

Wszystkie typy z DOM-a masz w https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts, możesz ich po prostu używać, są dostępne globalnie bez importowania. Zazwyczaj ich nazwa zaczyna się od `HTML`, np. do `<input>`a masz `HTMLInputElement`, a jeśli element jest dowolny, jest `HTMLElement`. IDE Ci powinno podpowiadać nazwę, gdy zaczniesz pisać.

Reply:
EZ dzięki ;)

Reply:
@mgol tylko po zmianach this.phoneInput.value = val; nie nadpisuje wartości w inpucie

Reply:
Jeśli tylko wprowadzisz inne typy, nie może to wpływać na runtime. Z Twojej wiadomości wynika jednak, że zmieniłeś też wartości. To nie `phoneInput` ma być `HTMLInputElement`, ale jego pole `nativeElement`.

Reply:
aaaa arigato